src/ast/passes/field_analyser.cpp

```diff
@@ -214,7 +214,7 @@ void FieldAnalyser::resolve_args(Probe &probe)
 
     auto probe_type = probetype(ap->provider);
     if (probe_type != ProbeType::fentry && probe_type != ProbeType::fexit &&
-        probe_type != ProbeType::uprobe)
```

Comment:
From what i could tell, having uprobe here was a mistake and didn't do anything.